cloud/gce/samples/3_node_cluster.tfvars

```diff
@@ -0,0 +1,35 @@
+# This configuration starts a 3-node CockroachDB cluster. You'll want to
```

Comment:
I don't think you need this file. `num_instances` will pretty much always be specified on the command line, and the others still need to be filled in anyway.